src/makers/hummingbird/hotspots.ts

```diff
@@ -0,0 +1,41 @@
+import HotspotIcon from './hummingbird.svg'
+import { MakerHotspot } from '../hotspotMakerTypes'
+import ANTENNAS from './antennas'
+
+const HUMMINGBIRD_H500 = {
+  name: 'Hummingbird H500',
+  icon: HotspotIcon,
+  onboardType: 'BLE',
+  translations: {
+    en: {
+      diagnostic:
+        '<b><white>Diagnostic support allows Hummingbird Maker to identify issues with your Hotspot in a secure way.</white></b>\n\nExample Maker will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@helium.com</b></purple> from the email used to purchase the Hotspot.',
```

Comment:
are you calling yourselves "Hummingbird Maker"? please also replace "Example Maker" with your name and replace the support email.

Reply:
fixed in new commit